udata/forms/fields.py

```diff
@@ -128,14 +129,6 @@ def process_formdata(self, valuelist):
             except ValueError:
                 self.data = None
                 raise validators.ValidationError(_('Invalid date/time input'))
-            except TypeError:
```

Comment:
Not needed anymore. Fixed in dateutil 2.3: https://bugs.launchpad.net/dateutil/+bug/1247643.